src/styles/app-theme/custom-mui.styles.ts

```diff
@@ -27,3 +27,15 @@ export const theme = createTheme({
     MuiTableRow: table
   }
 })
+
+export const breakpointsTheme = createTheme({
```

Comment:
Do you need to override mui default breakpoints, I think it doesn`t have a lot of sense

Reply:
> Do you need to override mui default breakpoints, I think it doesn`t have a lot of sense

removed breakpointsTheme. 
I already use AppDrawer for less screen sizes